src/ast.rs

```diff
@@ -264,7 +283,7 @@ impl Numb {
         } else if val.is_nan() {
           format!("+NaN")
         } else {
-          format!("{:?}", val)
```

Comment:
why this change?

Reply:
whoops, was testing to see if there were format specifiers where Rust and C both output the same float representation